presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -2954,6 +2954,7 @@ public void testIllegalStorageFormatDuringTableScan()
             String tableOwner = session.getUser();
             String schemaName = schemaTableName.getSchemaName();
             String tableName = schemaTableName.getTableName();
+            LocationService locationService = getLocationService();
```

Comment:
nit: perhaps, use `getLocationService()` in the next 2 lines and remove this variable